darkmoon/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+</body>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+
+    </div>
+
+    <h1>
+
+        <label for="username" class="opSystem" data-icon="u"></label>
```

Comment:
What are these fields being used for?

Reply:
It got duplicated by they link the seperate CSS file to the html. Without them there wouldn't be the CSS.